calm/dsl/cli/bp_commands.py

```diff
@@ -217,8 +217,19 @@ def create_blueprint_command(bp_file, name, description):
     default=False,
     help="Ignore runtime variables and use defaults",
 )
+@click.option(
+    "--input_runtime_editables",
+    "-I",
+    default=None,
+    help="Pass runtime editables via path to a json file or a json string",
+)
 def launch_blueprint_command(
```

Comment:
Also add another flag `--interactive/-i` , that will tell us to whether to prompt for input or not.
In any way the value present in the file provided will override existing data(defaults)..But if `--interactive` flag is true, U have to prompt for all the values.